server.js

```diff
@@ -15,19 +17,62 @@ import cors from 'cors'
 // overridden when starting the server. For example:
 //
 //   PORT=9000 npm start
-const port = process.env.PORT || 8080
-const app = express()
+const port = process.env.PORT || 8080;
+const app = express();
 
 // Add middlewares to enable cors and json body parsing
-app.use(cors())
-app.use(bodyParser.json())
+app.use(cors());
+app.use(bodyParser.json());
 
 // Start defining your routes here
-app.get('/', (req, res) => {
-  res.send('Hello world')
-})
+const endPoints = require('express-list-endpoints')
+app.get("/", (request, response) => {
+  response.send(endPoints(app));
+});
+
+// Get all the books in the list & filter options
+app.get('/books', (req, res) => {
```

Comment:
Try to be consistent - choose req, res or request, response and go with the same for all routes